tools/oesign/main.c

```diff
@@ -60,6 +61,9 @@ static const char _usage_sign[] =
     "                           certificate used to sign the digest file.\n"
     "  -d, --digest-file        path to the signed digest file matching the\n"
     "                           enclave image and specified configuration.\n"
+    "  -o, --output-file        [OPTIONAL] file name (with path) where the\n"
```

Comment:
We were using `optional` (lowercase) in other options. It would be great to keep the consistency :)

Reply:
Ah I missed it, I'll do that :-)